### DISCUSSION TITLE
Retention and Purging

### DISCUSSION THREAD

**Discussion by FriedCheese2006** [ep1cman/unifi-protect-backup#140] [Q&A] [answered]
Recently set this up. I have an LXC running debian in Proxmox with docker installed. I've gotten it setup to backup to Google Drive.

Just looking to make sure I understand this right. Default settings are 7 days for retention and 1 day for the purge check.

Does that mean any event over 7 days will be deleted from Google Drive during the purge job that's ran each day?

**Comment by ep1cman** [comment, marked as answer]
That's correct yes

**Reply by FriedCheese2006** [reply]
Small follow-on reply. I'm trying to shorten the retention.

```
docker run \
  -e UFP_USERNAME='XXXX' \
  -e UFP_PASSWORD='XXXX' \
  -e UFP_ADDRESS='XX.XX.XX.XX' \
  -e UFP_SSL_VERIFY='false' \
  -e RCLONE_DESTINATION='GD:/unifi_protect_backup' \
  -e DETECTION_TYPES='person' \
  -e RETENTION='3d' \
  -v '/root/clips':'/data' \
  -v '/root/rclone.conf':'/config/rclone/rclone.conf' \
  -v '/root/db':/config/database/ \
  ghcr.io/ep1cman/unifi-protect-backup
```

But at start, I'm not seeing it updating:

`2024-04-27 15:51:32 [   DEBUG   ] unifi_protect_backup.unifi_protect_backup_core :    retention=relativedelta(days=+7)`

**Reply by ep1cman** [reply]
please look at the readme file for the correct environment variable names

**Reply by FriedCheese2006** [reply]
Oooohhhhhh....I see it now. Thanks!
